components/google-cloud/google_cloud_componets/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Cloud Components root."""
+
+from google.cloud import aiplatform
+from .mb_sdk_component import utils
```

Comment:
Can you use absolute imports? go/pystyle#imports-decision

Reply:
done.